ehitchhiking-ui/src/app/shared/components/active-trip-additional-info/active-trip-additional-info.component.sass

```diff
@@ -0,0 +1,13 @@
+@import '../../../../styles/colors'
+@import '../../../../styles/mixins'
+
+mat-grid-tile
+  background: mat-color($accent, 50)
+.form-row
+  display: flex
```

Comment:
Could you please clarify why do you set display, justify-content, align-items and then use flexbox mixin? Is it possible to do it by using mixin only?

Reply:
O, yes. After creating the use of mixin forgot to remove